atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/keyvalue/impl/TableTasksTest.java

```diff
@@ -73,7 +73,7 @@ public void setup() {
         SweepStrategyManager ssm = SweepStrategyManagers.createDefault(kvs);
         Cleaner cleaner = new NoOpCleaner();
         SerializableTransactionManager transactionManager = new SerializableTransactionManager(
-                kvs, tsService, lockClient, lockService, txService, constraints, cdm, ssm, cleaner, false, 4);
+                kvs, tsService, lockClient, lockService, txService, constraints, cdm, ssm, cleaner, false, 8, 2);
```

Comment:
Why are these half of the defaults?

Reply:
No reason, will use the constants you mentioned above